contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1335,7 +1335,7 @@ fieldName: |-
   This is a string.
   And it can be on multiple lines.
 ----
-- When truncating YAML, always precede an ellipsis with a pound so that the YAML is valid.
+- When truncating YAML, the ellipsis should be commented out (`# ...`) because `...` is a link:https://yaml.org/spec/1.2.2/#22-structures[document end marker].
```

Comment:
I like the explanation, but I think it still should be direct on what to do. Suggesting something like:

```suggestion
- When truncating YAML, comment out the ellipses (`# ...`) because three dots (`...`) in YAML is actually a link:https://yaml.org/spec/1.2.2/#22-structures[document end marker].
```